app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
delete this file if you don't use it

Reply:
I am not sure this is necessary or not

Reply:
This file contain all helper method for your view, for now you don't have any method so you can delete it

Reply:
Noted it. Done

Reply:
Done